packages/vitest/src/node/logger.ts

```diff
@@ -303,6 +307,42 @@ export class Logger {
     this.log(c.red(divider()))
   }
 
+  getColumns() {
+    return 'columns' in this.outputStream ? this.outputStream.columns : 80
+  }
+
+  onExit(listener: Listener) {
```

Comment:
From the outside perspective, I am a bit confused by the `logger` is responsible for `onExit` callback 👀

Reply:
I like `onTerminalCleanup` 👍🏻